docs/source/llms/transformers/guide/index.rst

```diff
@@ -648,4 +678,156 @@ This input format requires that both the bitrate has been set prior to conversio
+
+* **Limited Databricks Integration**: If you are using Databricks, be aware that the model saved with `save_pretrained=False` cannot be registered to the legacy `Workspace Model Registry <https://docs.databricks.com/en/machine-learning/manage-model-lifecycle/workspace-model-registry.html>`_. If you want to register the reference-only Transformer model, please use `Unity Catalog <https://docs.databricks.com/en/machine-learning/manage-model-lifecycle/index.html>`_ instead, or download the model weight in advance using :py:func:`mlflow.transformers.persist_pretrained_model()` API as described in the next section.
+
+.. _persist-pretrained-guide:
+
+Persist the Model Weight to the Existing Reference-Only Model
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you want to update the reference-only model to the one contains the model weight, you can use :py:func:`mlflow.transformers.persist_pretrained_model()` API. This API will download the model weight from the HuggingFace Hub, save it to the artifact location, and update the metadata of the given reference-only model. After this operation, the model will be equivalent to the one saved with `save_pretrained=True` and be ready for the production use.
+
+.. code-block:: python
+
+    import mlflow
+    import transformers
+
+    pipeline = transformers.pipeline(
+        task="text-generation", model="databricks/dolly-v2-7b", torch_dtype="torch.float16"
+    )
+
+    # Save the reference-only Transformer model
+    with mlflow.start_run():
+        model_info = mlflow.transformers.log_model(
+            transformers_model=pipeline,
+            artifact_path="dolly",
+            save_pretrained=False,
+        )
+
+    # Model weight is not saved to the artifact store
+    assert not os.path.exists(model_info.artifact_path + "/model")
+
+    # This will download the model weight from the HuggingFace Hub and save it
+    # to the artifact location
+    mlflow.transformers.persist_pretrained_model(model_info.model_uri)
+
+    assert os.path.exists(model_info.artifact_path + "/model")
+
+
+PEFT Models in MLflow Transformers flavor
+-----------------------------------------
+
+.. warning::
+
+
+    The PEFT model is supported in MLflow 2.11.0 and above and is still in the experimental stage. The API and behavior may change in future releases. Moreover, the `PEFT <https://huggingface.co/docs/peft/en/index>`_ library is under active development, so not all features
+    and adapter types might be supported in MLflow.
+
+`PEFT <https://huggingface.co/docs/peft/en/index>`_ is a library developed by HuggingFace🤗, that provides various optimization methods for pretrained models available on the HuggingFace Hub. With PEFT, you can easily apply various optimization techniques like LoRA, QLoRA, to reduce the cost of fine-tuning Transformers models.
```

Comment:
```suggestion
`PEFT <https://huggingface.co/docs/peft/en/index>`_ is a library developed by HuggingFace🤗, that provides various optimization methods for pretrained models available on the HuggingFace Hub. With PEFT, you can easily apply various optimization techniques like LoRA and QLoRA to reduce the cost of fine-tuning Transformers models.
```
Can we add 1-2 sentences that give an overview of what this methodology means in the scope of "accelerated fine tuning" and how it can save applied GenAI developers a TON of time and headache compared with traditional fine tuning techniques? (speed, cost, accuracy, relevancy, hallucination reduction, etc...)

Reply:
Sounds great! Will add quick description of LoRA and how PEFT make it easy to use.